ui/imports/shared/views/PasswordView.qml

```diff
@@ -13,7 +13,10 @@ import StatusQ.Popups 0.1
 ColumnLayout {
     id: root
 
-    readonly property bool ready: newPswInput.text.length >= Constants.minPasswordLength && newPswInput.text === confirmPswInput.text && errorTxt.text === ""
+    readonly property bool ready: !d.isTooShort && // min length OK
+                                  !d.isTooLong && // max length OK
```

Comment:
This check was missing; two identical too long passwords would be considered "OK"